### ISSUE TITLE
Support for 12 and 24 word seed when initializing / recovering hw wallet

### ISSUE THREAD

**Issue by olemis** [skycoin/skycoin#2165]
**Feature description**

Support for hardware wallets with 24 words recovery seed .

**Is your feature request related to a problem? Please describe.**
For hardware wallets, desktop wallet does not provide controls to choose seed length and it can only generate 12 words mnemonic .

There is no way to choose at wallet initialization . Once it is detected ...
<img width="587" alt="screen shot 2019-02-13 at 2 43 32 pm" src="https://user-images.githubusercontent.com/538253/52739304-999c3400-2f9e-11e9-8dd2-afa433da0f85.png">

... clicking on `Configure automatically` only allows for confirmation.

<img width="583" alt="screen shot 2019-02-13 at 2 43 46 pm" src="https://user-images.githubusercontent.com/538253/52739418-eaac2800-2f9e-11e9-8977-c1c52c32b39f.png">

... and a control tracking operation progress.

<img width="581" alt="screen shot 2019-02-13 at 2 44 04 pm" src="https://user-images.githubusercontent.com/538253/52739672-61492580-2f9f-11e9-813c-a931df0b2510.png">

At a later moment , request for seed backup ...

<img width="581" alt="screen shot 2019-02-13 at 2 44 25 pm" src="https://user-images.githubusercontent.com/538253/52739482-0a435080-2f9f-11e9-9459-e00e52c2d60c.png">

... is immediately followed by instructions to read seed words on wallet screen .

<img width="582" alt="screen shot 2019-02-13 at 2 44 39 pm" src="https://user-images.githubusercontent.com/538253/52739577-38c12b80-2f9f-11e9-9435-7ef82be952c2.png">

**Describe the solution you'd like**
In my opinion (currently) the best approach would be to ask for seed length after clicking on `Create a backup` .

**Describe alternatives you've considered**
Maybe it's better to implement this after clicking `Configure automatically` button .

**Additional context**
Related to :
- skycoin/hardware-wallet#33 implemented by skycoin/hardware-wallet#70
- skycoin/hardware-wallet-go#23 implemented by skycoin/hardware-wallet-go#26
- skycoin/hardware-wallet-js#41 implemented by skycoin/hardware-wallet-js#49

Another suggestion would be to use consistent terminology in desktop wallet e.g. backup vs seed. This might be confusing for the regular user.

**Comment by olemis** [comment]
Now I noticed `seed` is used in message rendered in modal window to create backup . Maybe my last comment is not accurate .

**Comment by Senyoret1** [comment]
I have been doing some preliminary testing for this and I found a problem: when calling the `devGenerateMnemonic` function, no matter what value I enter, the seed always has 12 words.

I also saw that the `devRecoveryDevice` function respects the provided word count, but there is an adicional change: previously, when finishing entering all the words, the hardware wallet verified if the checksum of the seed was correct, now that does not happen. I'm not sure if that change is good.
